.github/workflows/integration-test.yml

```diff
@@ -11,38 +11,9 @@ jobs:
     name: Run integration tests
     runs-on: ubuntu-latest
     steps:
```

Comment:
We need to review the deletion of the manual setup of opensearch and dashboards.  This is the only way we can test pre-released features from the security backend or dashboards.

For more context see https://github.com/opensearch-project/security-dashboards-plugin/issues/886